apps/web/src/pages/api/price/index.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiHandler } from 'next'
+
+export const config = {
+  runtime: 'nodejs',
```

Comment:
shall we use `edge` runtime?

Reply:
having unknown error with it on deployment. will have to check more